vendors.d.ts

```diff
@@ -0,0 +1,44 @@
+import { FNOLCollectedAttributes } from "./fnol"
+
+export type Vendor = {
+  id?: string
+  deleted?: boolean
+  name?: string
+  website?: string
+  displayEmail?: string
+  notificationEmail?: string
+  displayPhone?: string
+  notificationPhone?: string
+  additionalProperties?: AdditionalProperties
+  handles?: HandlerOptions<FnolFilters>[]
+}
```

Comment:
I everything really optional?

If it's to make constructing the object easier, we could use some typing during construction that will do the same.

Reply:
Thats a good point.  The id/name are the only two that I think should be required.  It looks like I also need to add a vendor code which will be required.